crates/oxc_transformer/src/typescript/annotations.rs

```diff
@@ -417,29 +439,6 @@ impl<'a, 'ctx> Traverse<'a> for TypeScriptAnnotations<'a, 'ctx> {
             // Ignore ModuleDeclaration as it's handled in the program
             _ => true,
         });
```

Comment:
Do you think we to have a more general `Statement` helper to cover this usage?  We can mark a statement as `Delete` in `exit_statement`, this way we don't need to call `retain` each `exit_statements` @overlookmotel cc?

Reply:
Oh, you may need to open the file to look at the code